### DISCUSSION TITLE
How to upgrade hcloud-cloud-controller-manager and csi_driver

### DISCUSSION THREAD

**Discussion by slavus** [vitobotta/hetzner-k3s#386] [General]
What is recommended path to upgrade hcloud-cloud-controller-manager and csi_driver?

**Comment by vitobotta** [comment]
Hi, you can change the URLs for the manifests to point to newer ones and rerun the `create` command with hetzner-k3s, so it will upgrade them for you. BTW v2.0.0 is almost ready, there's is the rc1 available if you want to test it (see https://github.com/vitobotta/hetzner-k3s/discussions/385) :)
